test/integration/conftest.py

```diff
@@ -290,3 +290,46 @@ def create_vpc_with_subnet_and_linode(get_client, create_vpc_with_subnet):
     yield vpc, subnet, instance, password
 
     instance.delete()
+
+
+@pytest.fixture(scope="session")
+def create_vpc(get_client):
+    client = get_client
+
+    timestamp = str(int(time.time()))
```

Comment:
Let's use the nanosecond (`str(time.time_ns())`) to guarantee the label is truly unique.
And can we make this as a fixture similar to [this](https://github.com/linode/linodego/blob/proj/vpc/test/integration/util_test.go#L79-L81) for providing a unique label?